### ISSUE TITLE
Bitsquare won't open, can't connect to Tor

### ISSUE THREAD

**Issue by HSB65** [bisq-network/bisq#662]
I tried entering the Tor bridge address but Bitsquare won't open.

![tor1](https://cloud.githubusercontent.com/assets/18144690/19562761/d3cae61e-96dd-11e6-8b6c-f8e8bb910ab4.png)
![tor2](https://cloud.githubusercontent.com/assets/18144690/19562762/d3cb39c0-96dd-11e6-91ed-75a0f025b5c1.png)

**Comment by ManfredKarrer** [comment]
Please wait until the next version is out (today/tomorrow). Then click save and retry and quit the app. So the bridges get deleted, then normal tor should start up. Bridge support was inteded for GFW of China but did not help there...

**Comment by HSB65** [comment]
I have downloaded the new version Bitsquare-64bit-0.4.9.7
Still cannot start application, stuck at connecting to tor, so I cannot get the bridges deleted
Am attaching the log-file.

[bitsquare.txt](https://github.com/bitsquare/bitsquare/files/543517/bitsquare.txt)

**Comment by haiqu** [comment]
See also #639

**Comment by HSB65** [comment]
I looked at #639 . I didn't get any wiser. I still can't connect to Tor and Bitsquare won't launch.

**Comment by ManfredKarrer** [comment]
Do you have any funds, offers or trades? Otherwise if you never used Bitsquare before delete the data directory (of back it up just in case) and then start again. If it still shows the bridge page it means it cannot connect via Tor.
Then it seems a firewall problem maybe from your internet provider.
Usually Tor get over any firewalls, so far we only have issues with GFW of China.

**Comment by HSB65** [comment]
I do have funds, offers and trades.
As far as I can conclude, Tor is working. My ISP has blocked https://thepiratebay.org/ but I can reach it through Tor.

**Comment by ManfredKarrer** [comment]
Ok, then when u get the bridge screen leave it empty and click the save and retry button (you need to use the new release 0.4.9.7). After 5 sec. shut down the app and start new. If you still get to the bridge screen tor does not work in your environment. 
Does Tor browsers work for you?

**Comment by haiqu** [comment]
He did mention that Tor works for PirateBay. So it isn't broken apparently.

**Comment by HSB65** [comment]
Now finally I have managed to get it to work!

I am using the new release 0.4.9.7. The problem was that it was taking FOREVER to reach the bridge screen. Nothing seemed to happened so I closed the program. In the end I just decided to leave it running and then eventually after a very long time, Bitsquare launched. 

I followed Manfred's instructions and left the bridge screen empty and then I clicked the Save and retry button.  

There was not a problem with Tor because I could reach PirateBay.

Thanks for everyone's help.
